configs/tsup/src/index.ts

```diff
@@ -3,7 +3,7 @@ import type { Options } from 'tsup'
 export const options: Options = {
   banner: { js: '"use client"' },
   format: ['cjs', 'esm'],
-  // target: ['chrome51', 'es6', 'safari11', 'deno1', 'edge18', 'firefox53', 'node12'],
```

Comment:
Note to prevent misunderstanding: no target in current main. it's just test comment.
only check toDiff directory please.